app/globals.css

```
@@ -0,0 +1,70 @@
+}
+
+html,
+body {
+  height: 100%;
+  width: 100%;
+}
+
+body {
+  background-color: #f9f9f9;
+  color: #333;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```

Stylelint (17.10.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)